JustBrewIt/Pages/Brewery.cshtml

```diff
@@ -4,23 +4,29 @@
     var MyBreweryAPI = (QuickType.Welcome[])ViewData["MyBreweryAPI"];
 }
 @{
-    foreach (QuickType.Welcome welcome in MyBreweryAPI)
-    {   <table class="table table-bordered">
-    <thead>
-        <tr>
-            <th scope="col">Name</th>
-            <th scope="col">Brewery Type</th>
-            <th scope="col">Phone</th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr>
-            <td>@welcome.Name</td>
-            <td>@welcome.BreweryType</td>
-            <td>@welcome.Phone</td>
-        </tr>
-    </tbody>
-</table>
-
+    if (MyBreweryAPI.Any())
```

Comment:
Added a check if MyBreweryAPI has content.

Reply:
Good idea, though it was covered in a prior code review.  When that happens, first in wins.